guides/common/modules/proc_booting-virtual-machines.adoc

```diff
@@ -53,12 +53,19 @@ If you want to change the default values in the template, clone the template and
 [options="nowrap" subs="+quotes,attributes"]
 ----
 if exists user-class and option user-class = "iPXE" {
-  filename "http://{smartproxy-example-com}:8000/unattended/iPXE?bootstrap=1";
+  filename "http://{foreman-example-com}/unattended/iPXE?bootstrap=1";
 } # elseif existing statements if non-iPXE environment should be preserved
 ----
 +
 If you use an isolated network, use a {SmartProxyServer} URL with TCP port `8000`, instead of the URL of {ProjectServer}.
```

Comment:
Should work out of the box since 3.2, with the note that it's crucial that `--foreman-trusted-proxies` is used correctly on `{ProjectServer}. https://github.com/theforeman/foreman-documentation/pull/1626 has some of that, but it's not really great.

Reply:
Is this associated to a specific command we should mention?